dapp/src/components/ConnectWalletModal/ConnectWalletButton.tsx

```diff
@@ -239,7 +250,11 @@ export default function ConnectWalletButton({
                   Requires 2 actions:
                 </TextMd>
                 <ConnectWalletStatusLabel
-                  status={connectionStatus}
+                  status={
+                    connectionStatus === "idle" && reconnectStatus !== "idle"
+                      ? reconnectStatus
+                      : connectionStatus
```

Comment:
As I mentioned above we could have one status and include this logic in `useWallet`.

Reply:
aecbe9a